src/frontend/src/modules/SolarHomeSystem/LocationWidget.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <widget
+    :title="$tc('words.location')"
+    color="green"
+    :subscriber="'shs-location'"
+  >
+    <client-map
+      :mappingService="mappingService"
+      ref="mapRef"
+      :edit="false"
+      :zoom="12"
+    />
+  </widget>
+</template>
+
+<script>
+import Widget from "@/shared/widget"
+import ClientMap from "@/modules/Map/ClientMap.vue"
+import { MappingService, MARKER_TYPE } from "@/services/MappingService"
+
+export default {
+  name: "LocationWidget",
```

Comment:
In other components we don't really use Widget in the name. So, let's follow that convention and remove `...Widget` from here.